From aa6910ca37db66c51b7a9a6431b68f7223b8fd7b Mon Sep 17 00:00:00 2001
From: Radim Krcmar <rkrcmar@redhat.com>
Date: Wed, 27 Aug 2014 13:29:34 +0200
Subject: [PATCH 2/6] pc: increase maximal VCPU count to 240
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Message-id: <1409146174-28442-1-git-send-email-rkrcmar@redhat.com>
Patchwork-id: 60740
O-Subject: [RHEL7.1 qemu-kvm PATCH] pc: increase maximal VCPU count to 240
Bugzilla: 1134408
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Andrew Jones <drjones@redhat.com>

To prevent TCG from going over RHEL KVM's limit in bc949bc00c, we added
another downstream constraint that needs to be bumped.

Signed-off-by: Radim Krčmář <rkrcmar@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/i386/pc.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/include/hw/i386/pc.h b/include/hw/i386/pc.h
index 57e2f93..3b8f7d8 100644
--- a/include/hw/i386/pc.h
+++ b/include/hw/i386/pc.h
@@ -15,7 +15,8 @@
 #include "sysemu/sysemu.h"
 #include "hw/pci/pci.h"
 
-#define RHEL_MAX_CPUS 160
+/* KVM can't exceed KVM_SOFT_MAX_VCPUS (arch/x86/include/asm/kvm_host.h) */
+#define RHEL_MAX_CPUS 240
 
 /* PC-style peripherals (also used by other machines).  */
 
-- 
1.7.1

